packages/table-core/tests/unit/core/table/constructTable.test.ts

```diff
@@ -37,7 +38,6 @@ describe('constructTable', () => {
     expect(table).toHaveProperty('getRow')
 
     // table APIs
-    expect(table).toHaveProperty('_queue')
```

Comment:
@KevinVandy seems like _queue was removed.  just wanted to flag this to make sure I saw that correctly

Reply:
yeah, a couple of apis were removed